ee/clickhouse/queries/funnels/funnel_trends.py

```diff
@@ -117,7 +124,7 @@ def get_query(self) -> str:
             ) data
             RIGHT OUTER JOIN (
                 SELECT
-                    {interval_method}(toDateTime('{self._filter.date_from.strftime(TIMESTAMP_FORMAT)}') + number * {seconds_in_interval}) AS entrance_period_start
+                    {interval_method}(toDateTime('{formatted_date_from}') + number * {seconds_in_interval}) AS entrance_period_start
```

Comment:
Q: Why are we interpolating this instead of passing it as a param? 

While it seems safe here, it seems a safe assumption we'll mess up one day and we'll have sql injection on our hand.

Reply:
patched it here but we should do a check again through all the query building. I had originally removed most of the interpolating but I must've forgotten to emphasize it when all of us started building the new query classes